File: C_Programs/StoreNameInArray.cpp
//create a 2 d array to store 5 students name

#include<stdio.h>
#include<conio.h>

int main()
{
	char name[5][20];
	int i;
	for(i=0;i<5;i++)
	{	
		printf("\nEnter name %d : ",i);
		scanf("%s",&name[i]);	
	}
	for(i=0;i<5;i++)
	{
		printf("\n%s",name[i]);
	}
	return 0;
}

File: Basic_Java_Programs/StoreNameInArray.java
//create a 2 d array to store 5 students name
import java.util.*;

public class StoreNameInArray {
    public static void main(String[] args)
    {
        String[] name=new String[5];
        int i;
        Scanner sc=new Scanner(System.in);

        for(i=0;i<5;i++)
        {
            System.out.println("Enter name "+i+" : ");
            name[i]=sc.next();
        }
        for(i=0;i<5;i++)
        {
            System.out.println(name[i]);
        }
    }
}
